hoot-core/src/main/cpp/hoot/core/cmd/ConflateCmd.cpp

```diff
@@ -75,8 +76,22 @@ int ConflateCmd::runSimple(QStringList& args)
 
   if (args.contains("--differential"))
   {
+    // Setting these diff specific properties here on ConflateExecutor feels a little kludgy. May
+    // need some refactoring.
+
+    bool removeDiffLinearPartialMatchesAsWhole =
+      ConfigOptions().getDifferentialRemoveLinearPartialMatchesAsWhole();
+    if (ConflateUtils::isNetworkConflate() && !removeDiffLinearPartialMatchesAsWhole)
+    {
+      LOG_WARN(
+        "Removing elements partially from partial matches when using Differential Conflation is "
+        "not allowed when running the Network Algorithm.");
+      removeDiffLinearPartialMatchesAsWhole = true;
+    }
+
```

Comment:
Since the partial removal only works with Unifying, logging a warning here if someone tries to run with Network and forcing complete removal.